docs/preprocess-templates/expansion_logpoltens.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    modulate_amp_pol[mskmd]= b1
</pre><p>modulation of polarization is necessary because the log pol tens formalism doesn't give enough of it, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/galsci/pysm/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Yes, Giuseppe or @seclark can correct me, but I believe that a uniform modulation even in the poltens formalism was giving too little modulation in the plane and too much at high latitudes. The maps with the additional modulation looked _much_ more physical.

Reply:
Yes I confirm the modulation injected didn't yield enough modulation as we expected , basically it was uniform at different Galactic latitudes..